src/recon_buildblock/GeneralisedPrior.cxx

```diff
@@ -26,73 +26,14 @@
 
 START_NAMESPACE_STIR
 
-template <typename TargetT>
```

Comment:
as these were all moved now, the copyright header from this .cxx needs to be merged with the .inl (maybe nothing to do, I didn't check).

In fact, I wonder if we still need this .cxx

Reply:
This file is now essentially empty, aside from the instantiations. We don't really need those, so I think we can remove it.